package.json

```diff
@@ -48,6 +48,25 @@
     "url": "https://github.com/mdiehr/mtgnewsbot/issues"
   },
   "homepage": "https://github.com/mdiehr/mtgnewsbot#readme",
+  "dependencies": {
```

Comment:
well that's stupid. I guess it was moved and the diff didn't make it clear :hotdog: